css/SASS.scss

```diff
@@ -0,0 +1,1525 @@
+
+    .top-destinations__content{
+        display: flex;
+        gap: 35px;
+        margin-top: 60px;
+        flex-wrap: wrap;
+        position: relative;
+        justify-content: center;
+
+        & > .top-destinations__item:last-child::after{ 
+            position: absolute;
+            content: '';
+            background-image: url('../images/top-destination-decor.png');
+            background-repeat: no-repeat;
+            height: 254px;
+            width: 100px;
+            right: -65px;
+            bottom: +30px;
+            z-index: -10;
+
+            @include mq-max(462px){
+                display: none;
+            }
+        }
+    }
+
+    .top-destinations__item{
+        display: flex;
+        background-color: white;
+        
+        flex-direction: column;
+        
+        max-height: 460px;
+        max-width: 315px;
+        
+        box-shadow: 0px 100px 80px 0px rgba(0, 0, 0, 0.1);
+        border-radius: 24px;
+
+        position: relative;
+        transition: $animation-speed;
+        
+        &:hover{
+            .wrapped-image{
+                transform: scale(1.05);
+            }
+            cursor: pointer;
+        }
+
+        .top-destinations__picture-wrapper{
+            
+            position: relative;
+            flex-grow: 1;
+            overflow: hidden;
+            max-height: 330px;
+            border-radius: 24px 24px 0 0;
+            
+        
+            .wrapped-image{
+                width: 100%;
+                height: auto;
+                display: block;
+                transition: $animation-speed;
+            }
+        }
+
+        .top-destinations__price-set{
+            display: flex;
+            justify-content: space-between;
+            padding-top: 30px;
+            padding-right: 25px;
+            padding-left: 25px;
+
+            color: $text-color-general;
+
+            font-family: 'Poppins';
+            font-size: 18px;
+            font-style: normal;
+            font-weight: 500;
+            line-height: 22px;
+            letter-spacing: 0em;
+        }
+
+        .top-destinations__days{
+            font-family: 'Poppins';
+            font-size: 16px;
+            font-style: normal;
+            font-weight: 500;
+            line-height: 20px;
+            letter-spacing: 0em;
+            text-align: left;
+
+            padding-top: 20px;
+            padding-right: 25px;
+            padding-left: 25px;
+
+            color: $text-color-general;
+            position: relative;
+
+            padding-bottom: 42px;
+
+            &::before{
+                left: 0;
+                content: '';
+                background-image: url('../images/gps-icon.svg');
+                background-repeat: no-repeat;
+                width: 17px;
+                height: 17px;
+                top: 3px;
+                display: inline-block;
+                margin-right: 15px;
+            }
+        }
+    }
+}
+
+.book-trip{
```

Comment:
Remove the inheritance of independent selectors throughout the code. It is hard to review and modify code. If you look into your CSS code, you see a long hard read "sausages" which are not appropriate by BEM